crowbar_framework/app/assets/stylesheets/variables.scss

```diff
@@ -157,26 +158,20 @@ $legend-border-color: #e5e5e5;
 $input-group-addon-bg: $gray-lighter;
 $input-group-addon-border-color: $input-border;
 
-
 // Dropdowns
 // -------------------------
-$dropdown-bg: #25282a;
-$dropdown-border: rgba(0, 0, 0, 0.2);
-$dropdown-fallback-border: #ccc;
-$dropdown-divider-bg: #25282a;
-
-$dropdown-link-color: #bebebe;
-$dropdown-link-hover-color: $nav-font-color;
-$dropdown-link-hover-bg: #2e3235;
-
-$dropdown-link-active-color: #fff;
-$dropdown-link-active-bg: #25282a;
-
-$dropdown-link-disabled-color: $gray-light;
-
+$dropdown-bg: #fff !default;
+$dropdown-border: rgba(0,0,0,.15);
```

Comment:
`.15` should be written with a leading zero as `0.15`<br>Commas in function arguments should be followed by a single space